.github/workflows/build_test.yml

```diff
@@ -0,0 +1,27 @@
+name: Build & Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Install Python Dependencies
+        run: pip install -r requirements.txt
+
+      - name: Install Dependencies
+        run: yarn
+
+      - name: Build
+        run: yarn build
```

Comment:
So far I have followed tutorials that use npm not yarn, and don't have yarn installed on the prod server yet.